src/mini-graphql-http-client.js

```diff
@@ -50,6 +50,7 @@ const nativeFetch = typeof fetch === "undefined" ? null : fetch;
  * @param uri {String} The GraphQL HTTP endpoint. Like https://example.com/graphql
  * @param [method="POST"] {String} the HTTP method for all the request. Default it "POST"
  * @param [fetch] {Function} Fetch implementation. You must provide it if global `fetch` is missing.
+ * @param retry {Number} Numbers of retry attempts on 5xx response or network errors
```

Comment:
`retry` is optional. Thus must be in square brackets: `[retry]`
See more examples here: https://devhints.io/jsdoc

Reply:
The default value is missing from JSDoc.